witnet/GlintUsdtPrice.js

```diff
@@ -0,0 +1,50 @@
+import * as Witnet from "witnet-requests"
+
+// Retrieves GLINT/USDT-6 from the Hoo HTTP-GET API
+const hoo = new Witnet.Source("https://api.hoolgd.com/open/innovate/v1/tickers/market")
```

Comment:
This data source is timing out for me!

Reply:
yep, that's why it's been disabled on all chains; the javascript source code remains in the repo just in case Hoo API (only source atm) comes back to live